### PR TITLE
Bump react-native-win32 and add pressable text test

### PR DESCRIPTION
**Pull request by ksiler**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [x] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Bump react-native-win32 version which introduces key events, selectable, and tooltip to Text.

Add pressable text test (win32 only)

### Verification

(how the change was tested, including both manual and automated tests)

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: apps/fluent-tester/package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "@fluentui-react-native/themed-stylesheet": "^1.1.0",
     "@fluentui-react-native/win32-theme": ">=0.5.1 <1.0.0",
     "@fluentui/react-native": ">=0.21.7 <1.0.0",
+    "@office-iss/react-native-win32": "0.62.6",
```
Comment (tkamireh) [inline apps/fluent-tester/package.json:40]: probably want to hat this to "^0.62.6", idk why it wasn't like that before

File: apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { Alert, Linking, Platform, View } from 'react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import { Text } from '@fluentui/react-native';
+import { stackStyle } from '../Common/styles';
+import { IKeyboardEvent, IHandledKeyboardEvent } from '@office-iss/react-native-win32';
+
+export const PressableUsage: React.FunctionComponent<{}> = () => {
+  if (Platform.OS == ('win32' as any)) {
```
Comment (tkamireh) [inline apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.tsx:9]: Not strictly necessary to platform guard is it? keyDown just won't do anything on the other platform, worth adding a comment perhaps, maybe even a platform check in dev mode inside the Text component itself that logs a warning or error.

File: apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      if (ev.nativeEvent.key === 'Enter' || ev.nativeEvent.key == ' ') Alert.alert('Alert', 'Success!');
+    };
+
+    const handledNativeKeyboardEvents: IHandledKeyboardEvent[] = [{ key: 'Enter' }, { key: 'Space' }];
+
+    return (
+      <View>
+        <Stack style={stackStyle} gap={5}>
+          <Text variant={'bodyStandard'}>
+            To learn more about microsoft, visit this{' '}
```
Comment (tkamireh) [inline apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.tsx:32]: What does the rendered output look like? Is there a formatting thing I'm missing?

File: apps/win32/package.json [hunk 4]
```diff
@@ -60,7 +60,7 @@
     "@wdio/sync": "5.20.1"
   },
   "peerDependencies": {
-    "@office-iss/react-native-win32": "0.62.3",
+    "@office-iss/react-native-win32": "0.62.6",
```
Comment (tkamireh) [inline apps/win32/package.json:63]: You may be able to get rid of this, it's not really a peer dep. Not sure it's a dev dep either, though it seems like some things may have broken when removing it from dev deps and placing it as a dep in other places? Any idea what was going on there?

File: apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.tsx (hunk 3 above)
Comment (ksiler) [reply apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.tsx:32]: Are you referencing the `{' '}`? It's just a trailing space on the first part of the sentence. It looks like a regular sentence with an inline link.

File: packages/components/Button/package.json [hunk 5]
```diff
@@ -31,7 +31,7 @@
     "@uifabricshared/foundation-settings": ">=0.8.0 <1.0.0"
   },
   "devDependencies": {
-    "@office-iss/react-native-win32": "0.62.3",
+    "@office-iss/react-native-win32": "0.62.6",
```
Comment (tkamireh) [inline packages/components/Button/package.json:34]: Like the other comment, these probably should be deps (unless we only use the typings and want to keep them opaque)

**Review by tkamireh — APPROVED** [review]
Generally looks good, left some questions

File: apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.tsx (hunk 3 above)
Comment (tkamireh) [reply apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.tsx:32]: yes, nice. Text components don't render trailing whitespace? For a second, I thought there was some crazy template literal type stuff going on

File: apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.tsx (hunk 2 above)
Comment (Saadnajmi) [reply apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.tsx:9]: I know that @HeyImChris recently added keyboarding to react-native-macos so maybe it does? He would be better to comment though

File: apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.tsx (hunk 3 above)
Comment (ksiler) [reply apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.tsx:32]: I think they do, it's just added in by the editor

File: apps/win32/package.json (hunk 4 above)
Comment (ksiler) [reply apps/win32/package.json:63]: No idea. I just tried it locally and it seems to be fine. I'll move it to dependencies.

File: apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.tsx (hunk 2 above)
Comment (ksiler) [reply apps/fluent-tester/src/FluentTester/TestComponents/Text/PressableUsage.tsx:9]: I see that, looks like it works slightly differently than it does for win32 though. I can check to see if it causes any issues.

**Comment by tkamireh** [comment]
Looks like in fluent-tester-win32, react-native-win32 isn't used as a dep (we should just remove  it from that package).

File: apps/win32/package.json (hunk 4 above)
Comment (ksiler) [reply apps/win32/package.json]: Actually just started seeing problems now. When removing it, the win32 package will no longer bundle. Adding it back to devDependencies fixes it.

**Pull request merged**